src/demo/grasp_filter_demo.cpp

```diff
@@ -119,37 +120,44 @@ class GraspFilterDemo
 
     // ---------------------------------------------------------------------------------------------
     // Load grasp data specific to our robot
-    grasp_data_ = std::make_shared<moveit_grasps::GraspData>(nh_, ee_group_name_, visual_tools_->getRobotModel());
+    grasp_data_ =
+        std::make_shared<moveit_grasps::TwoFingerGraspData>(nh_, ee_group_name_, visual_tools_->getRobotModel());
+    if (!grasp_data_->loadGraspData(nh_, ee_group_name_))
+    {
+      ROS_ERROR_STREAM_NAMED(LOGNAME, "Failed to load Grasp Data parameters.");
+      exit(-1);
+    }
 
     // ---------------------------------------------------------------------------------------------
     // Clear out old collision objects
     visual_tools_->removeAllCollisionObjects();
 
     // ---------------------------------------------------------------------------------------------
     // Load grasp generator
-    grasp_generator_ = std::make_shared<moveit_grasps::GraspGenerator>(visual_tools_);
+    grasp_generator_ = std::make_shared<moveit_grasps::TwoFingerGraspGenerator>(visual_tools_);
 
     // Set the ideal grasp orientation for scoring
     std::vector<double> ideal_grasp_rpy = { 3.14, 0.0, 0.0 };
```

Comment:
Use M_PI here?

Reply:
You're definitely right but it's a demo so I don't feel the need to make this change here.